core/templates/components/concept-card/concept-card.directive.ts

```diff
@@ -73,6 +73,9 @@ angular.module('oppia').directive('conceptCard', [function() {
             }
             // TODO(#8521): Remove when this directive is migrated to Angular.
             $rootScope.$apply();
+          }, function(errorResponse) {
+            ctrl.loadingMessage = 'Oops, it looks like this skill has been ' +
```

Comment:
Actually, it seems odd to have this message assigned to "loadingMessage", right? Shoul d we use a more appropriate variable name for this? Also, I just noticed that we show the "..." (loading dots) in the screenshot. Perhaps we should not have those.

Reply:
Done, PTAL.